contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,156 @@
+
+
+# Kubernetes Cluster with Containerd and Cri-Containerd
+
+This document provides the steps to bring up a Kubernetes cluster using ansible and kubeadmusing ansible and kubeadm tools. 
+
+### Prerequisites:
+- **OS**: Ubuntu 16.04 (will be updated with additional distros after testing)
```

Comment:
Please express that it's welcome to validate this in other environments and file issue, or contribute support for more platforms here.

Reply:
I think you've mentioned this at the end.